src/Lib/Lionk.Notifications/Model/Abstraction/IChannel.cs

```diff
@@ -35,4 +44,11 @@ public interface IChannel
     /// <param name="notifyer"> The notifyer that send the notification.</param>
     /// <param name="content"> The content of the notification.</param>
     void Send(INotifyer notifyer, Content content);
+
+    /// <summary>
+    /// Method to compare two objects.
+    /// </summary>
+    /// <param name="obj"> The object to compare.</param>
+    /// <returns> True if the objects are equals, otherwise false.</returns>
+    bool Equals(object? obj);
```

Comment:
I think that the best way to force Equals implementation is by implementing IEquatable<T>.
So IChannel must inherit from IEquatable.

As you want i'm not sure

Reply:
yes, good